architecture_docs/plantuml/03_context_view.puml

```diff
@@ -0,0 +1,25 @@
+@startuml
+
+title Loculus Context View
+left to right direction
+
+actor User as user
+actor Submitter as submitter
+actor INSDC as insdc
+
+component Loculus as loculus {
+    portin " " as download
+    portin " " as analysis
+    portin " " as upload
+    portout " " as ingest
+    portout " " as deposition
+}
+
+user <-- download : Download data
+user <-- analysis : Query data for analysis
+submitter --> upload : Upload data
+
+insdc -down-> ingest : Download data from NCBI
+insdc <-up- deposition : Upload data to ENA
```

Comment:
I wonder whether we should be a bit more general here because it is within the supported scope of Loculus that other people can develop own ingest and upload pipelines for other databases. But the current verison is also alright for now and we can adapt later.

```suggestion
external databases (e.g., INSDC) -down-> ingest : Download data
external databases (e.g., INSDC) <-up- deposition : Upload data
```